lib/conduit_lwt_server.mli

```diff
@@ -1,6 +1,8 @@
 
 val close : 'a Lwt_io.channel * 'b Lwt_io.channel -> unit Lwt.t
 
+val set_max_active : int -> unit
```

Comment:
I see, that makes sense. But although it's a global resource we can only share this limit between conduit servers here.  

Anyway, I say let's leave it as is for now. If we need more flexibility, then we can consider your proposal here. Seems like adding this limit was motivated by a concrete need you had so let's wait until something concrete like that pops up.